src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -47,12 +47,14 @@ List<Activity> activities = (List<Activity>) request.getAttribute("activities");
 
     <div class="container">
       <h2 style="color:blue; text-aligned:left; margin-bottom:25px"> ACTIVITY </h2>
-      <h3> Here's everything that's happened on the site so far! </h3>
-        <div id="activityfeed">
-          <ul>
-            <%
-              for (int i = activities.size() -1; i>-1; i--) {
-                Activity activity = activities.get(i);
+      <% if (activities == null || activities.size() == 1) { %>
+      <h3> Nothing happened on the site so far! </h3>
+      <% } else { %>
```

Comment:
If there's 1 activity why do we want to say that nothing happened? Is this because the admin is added?